dts/bindings/sensor/adi,adltc2990.yaml

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) 2023 Carl Zeiss Meditec AG
+# SPDX-License-Identifier: Apache-2.0
+
+
+description: ADLTC2990 Quad I2C Voltage, Current and Temperature Monitor
+
+compatible: "adi,adltc2990"
+
+include: [sensor-device.yaml, i2c-device.yaml]
+
+properties:
+  mode_v1_v2:
+    type: int
+    required: true
+    description: |
+      Define how these channels should be used
+      nothing = 0
+      current = 1
+      differential = 2
+      single-ended = 3
+      temperature = 4
+    enum:
+      - 0
+      - 1
+      - 2
+      - 3
+      - 4
+
+  pins_v1_v2_current_resistor:
+    type: int
+    description: Define the resistor to be used for measuring current in microohms
+    default: 1000
```

Comment:
Defaults need justification in the description. Is it a register reset value?

Reply:
In order to measure the current from voltage differenece of V1 and V2, some resistor would be there. And this is that resistor. I have just added some default value of 1 milliohms, but what would be the best way to represent this resistor?